client/trino-jdbc/src/main/java/io/trino/jdbc/TrinoDriverUri.java

```diff
@@ -467,7 +467,7 @@ private static void validateConnectionProperties(Properties connectionProperties
     }
 
     @VisibleForTesting
-    static void setRedirectHandler(RedirectHandler handler)
+    public static void setRedirectHandler(RedirectHandler handler)
```

Comment:
Methods visible for testing should be at most package private. This is especially true in a library where users may inadvertently become dependent on a public method, and JDBC is a library.